client/main.c

```diff
@@ -2,8 +2,11 @@
 
 #include "init.h"
 
+#include "utils/utils.h"
 int main(int argc, char **argv)
 {
 	xdag_init(argc, argv, 0);
+//	char a[] = "123456789012345678901234567890abcde";
```

Comment:
Do you need these commented lines?

Reply:
Oh, this file change should not be submitted.